discordrb.json

```diff
@@ -57,6 +57,28 @@
             "voiceReceive": "no",
             "voiceSend": "yes",
             "voiceStateUpdate": "yes"
+        },
+        "misc": {
+            "userLoginWithPassword": "yes",
```

Comment:
Can we get some more opinions on whether this should be included? Has anyone removed or have plans to remove user logins from their libraries?